.circleci/config.yml

```diff
@@ -271,10 +299,49 @@ jobs:
           command: |
             chamber exec cosmos-sdk -- start_website_build
 
+  macos_ci:
+    <<: *macos_defaults
+    steps:
+      - *macos_env
+      - run:
+          name: Install go
+          command: |
+            source $BASH_ENV
+            curl -L -O https://dl.google.com/go/go$GO_VERSION.darwin-amd64.tar.gz
+            tar -C $HOME -xzf go$GO_VERSION.darwin-amd64.tar.gz
+            rm go$GO_VERSION.darwin-amd64.tar.gz
+            go version
+      - checkout
+      - run:
+          name: Install SDK
+          command: |
+            source $BASH_ENV
+            make get_tools
+            make get_vendor_deps
+            make install
+            make install_examples
+      - run:
+          name: Integration tests
+          command:
+            source $BASH_ENV
+            make test_cli
+            make test_examples
```

Comment:
We don't need to `test_examples` anymore

Reply:
Removed.